### PR TITLE
pavucontrol, waybar: Use system style

### PR DESCRIPTION
**Pull request by Flakebi**
###### Motivation for this change
pavucontrol did not respect the system gtk style.
Same for waybar, the tray used the default icons instead of the selected icon theme.

###### Things done
Adding `wrapGAppsHook` fixes the style and icons for pavucontrol and waybar.

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc

### CODE REVIEW

File: pkgs/applications/audio/pavucontrol/default.nix [hunk 1]
```diff
@@ -19,7 +19,7 @@ stdenv.mkDerivation rec {
   buildInputs = [ libpulseaudio gtkmm3 libcanberra-gtk3 makeWrapper
```
Comment (jtojnar) [inline pkgs/applications/audio/pavucontrol/default.nix]: You should remove `makeWrapper` and `preFixup` since that is already handled by `wrapGAppsHook`.

Comment (Flakebi) [reply pkgs/applications/audio/pavucontrol/default.nix]: Thanks, done

**Comment by kisik21** [comment]
Wow, never noticed it in pavucontrol but definitely wondered why doesn't Waybar use my icons. (Used it in the past) I think I could find another app or two that don't use icons correctly on my machine.

**Review by vikanezrimaya — APPROVED** [review]

**Comment by jtojnar** [comment]
Could you describe the system you are running (DE/WM/possibly a config)? I do not understand why should this be necessary if you already have an icon theme installed – GTK picks up themes from `XDG_DATA_DIRS`, so when you have an icon theme in `environment.systemPackages`, it should find it.

**Comment by Flakebi** [comment]
Any news on this?
I’m using the patched packages since a few months now and didn’t notice issues.
So, can it be merged?

Thanks!

**Pull request merged**